### PR TITLE
Java debug tools. Jstack, jmap and jhat.

### PR DESCRIPTION
**Pull request by notpeter**

### CODE REVIEW

File: pages/common/jstack.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> Java Stack Trace Tool.
+
+- Print java stack traces for all threads in a java process:
+
+`jstack {{java_pid}}`
+
+- Print mixed mode (java/c++) stack traces for all threads in a java process:
+
+`jstack {{java_pid}}`
```
Comment (ostera) [inline pages/common/jstack.md]: is this the exact same command as before?

File: pages/common/jmap.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+> Java Memory Map Tool.
+
+- Print shared object mappings for a java process (output like pmap):
+
+`jmap {{java_pid}}`
+
+- Print heap summary information:
+
+`java -heap {{filename.jar}} {{java_pid}}`
```
Comment (ostera) [inline pages/common/jmap.md]: This is not the `jmap` command?

**Comment by notpeter** [comment]
@ostera Nice catch.   Fixed both those in a subsequent commit.

**Comment by ostera** [comment]
Thanks @notpeter

**Pull request merged**